app/assets/javascripts/tagging.js

```diff
@@ -1,10 +1,15 @@
 function addTag(tagname, selector) {
 
   selector = selector || '#tagform';
+  if (tagname.indexOf("place") != -1) {
+    place = tagname.split(":")[1];
+    place.replace("-", " ");
+    alert("This looks like a location");
+  }
 
   var el = $(selector);
 
-  el.find('.tag-input').val(tagname);
+  el.find('.tag-input').val("tagname");
```

Comment:
I think the quotes are not needed - we want the variable -- but, we could put this in an `else` clause after checking if it's a place. So:

1. if it's a place, start the dialog box and wait until a) we geocode and b) user says OK or Cancel
2. if it's not, just go ahead and submit the tag!

Make sense? 

Good going Mridul, exciting to see this start off!

Reply:
Got it, on it!